test/zombieTest.js

```diff
@@ -0,0 +1,22 @@
+const Browser = require('zombie');
+
+import server from '../server.js';
+
+describe('User visits landing page', function(){
+	const browser = new Browser();
+
+
+	before(function(done){
+		server.listen(8080, () => {
```

Comment:
nitpick: this can now be onelined: `server.listen(8080, () => browser.vist(...)`